src/modals/contactMomenten/ContactMomentenForm.vue

```diff
@@ -3,19 +3,39 @@ import { contactMomentStore, navigationStore, taakStore, zaakStore } from '../..
 </script>
 
 <template>
-	<NcDialog name="Contactmoment"
+	<NcDialog :name="Object.keys(contactMomenten).length > 1 ? 'Contactmomenten' : 'Contactmoment'"
 		size="large"
 		label-id="contactMomentenForm"
 		dialog-classes="ContactMomentenForm"
 		:close-on-click-outside="false"
 		@closing="closeModalFromButton()">
 		<div v-if="!isView" class="tabContainer">
-			<BTabs card>
+			<div class="newTabButtonContainer">
+				<NcButton type="primary" class="newTabButton" @click="() => newTab()">
+					<template #icon>
+						<Plus :size="20" />
+					</template>
+					Nieuw contactmoment
+				</NcButton>
+			</div>
```

Comment:
The Button works really nice, but it does not remove the contactmoment from the array.
Which in turn does not allow changing the name `Contactmomenten` back to `Contactmoment`